pandas/core/internals/blocks.py

```diff
@@ -2185,6 +2176,7 @@ def fillna(
         )
 
     def quantile(self, qs, interpolation="linear", axis: int = 0) -> Block:
+        assert axis == 1  # only ever called this way
```

Comment:
prob should just change the default? also can you type qs now / assert it?

Reply:
i see you are doing that below already.

Reply:
> type qs now

tried annotating and got unhelpful mypy complaints